debug/gdbdiff.ts

```diff
@@ -23,25 +67,29 @@ function compareRegisters(emulator: Uint32Array, silicone: Uint32Array) {
 async function main() {
   const client1 = new GDBClient();
   const client2 = new GDBClient();
-  await client1.connect('localhost');
-  await client2.connect('raspberrypi');
+  await client1.connect('localhost', 3334);
+  await client2.connect('localhost', 3333);
   // Disable interrupts
   await client1.writeRegister(19, 1, 8);
   await client2.writeRegister(19, 1, 8);
   // Start diffing
-  for (let counter = 0; ; counter++) {
+  let prevRegSet = await client2.readRegisters();
+  let counter = 1;
```

Comment:
No need to separately delcare `counter`, as it is only used inside the `for` loop.

We can move the `let` to the for statement, e.g. `for (let counter = 1; ; counter++) {`